src/components/Modal/Strategy.vue

```diff
@@ -68,40 +87,48 @@ watch(open, () => {
       :placeholder="$t('searchPlaceholder')"
       :modal="true"
     />
+
     <div v-if="input.name" class="m-4">
-      <SDefaultObject
-        v-if="definition"
-        v-model="input.params"
-        :definition="definition"
-      />
-      <UiButton
-        v-else
-        class="block w-full mb-3 overflow-x-auto"
-        style="height: auto"
-      >
-        <TextareaJson
+      <RowLoading v-if="loading" class="px-0" />
+      <div v-else>
+        <SDefaultObject
+          v-if="definition"
           v-model="input.params"
-          v-model:is-valid="isValid"
-          :placeholder="$t('strategyParameters')"
-          class="input text-left no-scrollbar"
-          style="width: 560px"
+          :definition="definition"
         />
-      </UiButton>
+        <UiButton
+          v-else
+          class="block w-full mb-3 overflow-x-auto"
+          style="height: auto"
+        >
+          <TextareaJson
+            v-model="input.params"
+            v-model:is-valid="isValid"
+            :placeholder="$t('strategyParameters')"
+            class="input text-left no-scrollbar"
+            style="width: 560px"
+          />
+        </UiButton>
+      </div>
     </div>
-    <div v-else class="my-4 mx-0 md:mx-4">
-      <a
-        v-for="strategy in strategiesResults"
-        :key="strategy.key"
-        @click="handleClick(strategy)"
-      >
-        <BlockStrategy :strategy="strategy" />
-      </a>
-      <NoResults v-if="Object.keys(strategiesResults).length < 1" />
+
+    <div v-else class="my-4 mx-0 md:mx-4 min-h-[339px]">
+      <RowLoadingBlock v-if="loadingStrategies" />
+      <div v-else>
+        <a
+          v-for="strategy in strategiesResults"
+          :key="strategy.id"
+          @click="selectStrategy(strategy.id)"
+        >
+          <BlockStrategy :strategy="strategy" />
+        </a>
+        <NoResults v-if="Object.keys(strategiesResults).length < 1" />
```

Comment:
`strategiesResults` is already an array. You removed `Object.keys` [here](https://github.com/snapshot-labs/snapshot/pull/1583/files#diff-ff499283d92fd05a331536bdc3d1a30cccc47cfb5ca2c8f6b61d729ca2dc7f63R128) too.

```suggestion
        <NoResults v-if="strategiesResults.length < 1" />
```